src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -28,6 +28,11 @@
         <import resource="services/validator.xml" />
     </imports>
 
+    <parameters>
+        <parameter key="sylius.model.address.class.api.denormalize">%sylius.model.address.class%</parameter>
+        <parameter key="sylius.model.address.interface.api.denormalize">Sylius\Component\Core\Model\AddressInterface</parameter>
```

Comment:
```suggestion
        <parameter key="sylius.model.address.interface.api.denormalize">%sylius.model.address.interface.class%</parameter>
```

Therfore we could even remove this custom parameters, as we have everything from Resource. The drawback of this solution is that, we will need to make typehints to the first class in the interface chain. On the contrary, it will work exactly the same as in Doctrine Mapping.